packages/e2e-tests/src/config/bootstrap.js

```diff
@@ -214,31 +214,6 @@ function observeConsoleLogging() {
   });
 }
 
-/**
- * Runs Axe tests when the story editor is found on the current page.
- *
- * @return {?Promise} Promise resolving once Axe texts are finished.
- */
-async function runAxeTestsForStoriesEditor() {
```

Comment:
We need to look at doing an a11y test run or something vs having this run after `afterEach`